oscar_elasticsearch/search/search_handlers.py

```diff
@@ -32,16 +32,17 @@ def items(self):
 class SearchHandler(object):
     model = None
     form_class = BaseSearchForm
+    suggestion_field_name = "title"
```

Comment:
Moet dit geen settings zijn?

Reply:
no if you want to change it you can override it, it is very unlikely that you want something else for this setting